README.md

```diff
@@ -53,3 +54,11 @@ Feel free to raise and fix issues.
 For any questions, join #oshc-dev on Slack. Get an invite [here](https://opensourcehelp.herokuapp.com/).
 
 Note : All design related tasks have reward associated with them.
+
+
+<!-- develop -->
```

Comment:
@Logan1x I'm sorry but I didn't really look at the PR before the last few commits were added so I am not aware of what _exactly_ @tapasweni-pathak meant.

But what happens in conflicts is usually this: you submit a PR/change. Meanwhile, before this PR is merged, someone else changes the related part of the code-base.

Now if I merge your PR, it would create a **conflict** i.e. git won't understand whether to use your code or the other changed code. To resolve the conflict, you need to do something like this:

```bash
$ git fetch upstream
$ git rebase upstream/develop

# this will tell you that there are conflicts and you must solve them yourself.
# this is where you decide which changes to take in i.e. you'll see the
# >>>>>>>> lines.
```

I'm guessing you already went through this part before? If not and if you find this confusing (which it can be), let me know and we'll figure it out together. :)

So yeah, I don't think @tapasweni-pathak meant adding the comment; just resolving the conflicts - you don't need to add a comment to fix the conflicts.